docker-nlp-toolset/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM maluuba/tomcat7
+MAINTAINER Maluuba NLP Team <nlp@maluuba.com>
+
+RUN apt-get install -qqy git docker.io
+
+# Maven
+RUN apt-get remove maven2
+ENV MAVEN_VERSION 3.2.5
+RUN curl -sSL http://archive.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar xzf - -C /usr/share \
+  && mv /usr/share/apache-maven-$MAVEN_VERSION /usr/share/maven \
+  && ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
+ENV MAVEN_HOME /usr/share/maven
+
+RUN mvn -version
```

Comment:
This will create a diff without a good reason its probably better to skip this. You would only see this log at build time anyway